docs/2023/sunbird-framework-management-ui-tool/index.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 1
+title: sunbird-framework-management-ui-tool
+slug: /2023/Sunbird Knowlg/sunbird-framework-management-ui-tool
+---
+
+
+# Author
+katakam pranavi
+
+
+# project name
+sunbird-framework-management-ui-tool
+
+
+# Contact Info
+[Email](mailto:pranavi123katakam@gmail.com)
+[Linked In](https://www.linkedin.com/in/pranavi-katakam35)
```

Comment:
I have edited this link as this was incomplete. Please take a note for next time